electrum/gui/qt/confirm_tx_dialog.py

```diff
@@ -426,6 +426,7 @@ def add_cv_action(configvar: 'ConfigVarWithConfig', action: Callable[[], None]):
         add_cv_action(self.config.cv.WALLET_MERGE_DUPLICATE_OUTPUTS, self.toggle_merge_duplicate_outputs)
         add_cv_action(self.config.cv.WALLET_SPEND_CONFIRMED_ONLY, self.toggle_confirmed_only)
         add_cv_action(self.config.cv.WALLET_COIN_CHOOSER_OUTPUT_ROUNDING, self.toggle_output_rounding)
+        add_cv_action(self.config.cv.WALLET_FREEZE_REUSED_ADDRESS_UTXOS, self.toggle_freeze_reused_address_utxos)
```

Comment:
Unclear if the TxEditor is the best place to put this.
We could put it into the utxo_list toolbar menu instead.
(or both)